microprofile-openapi/README.adoc

```diff
@@ -20,28 +20,15 @@ To complete this guide, you will need:
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven 3.5.3+
 
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+ 
 == Steps
 
-. Start a {productName} instance.
+. Start a {productName} instance using the microprofile profile.
 +
 [source, shell]
 ----
-$ mvn wildfly:run &
-----
-
-. Add support for MicroProfile OpenAPI by adding the requisite extension and subsystem.
-+
-[source, shell]
-----
-$ mvn wildfly:execute-commands -Dwildfly.commands=/extension=org.wildfly.extension.microprofile.openapi-smallrye:add
-$ mvn wildfly:execute-commands -Dwildfly.commands=/subsystem=microprofile-openapi-smallrye:add
-----
-
-. Since the MicroProfile OpenAPI subsystem modifies {productName}'s deployment chain, the server needs to be reloaded.
-+
-[source, shell]
-----
-$ mvn wildfly:shutdown -Dwildfly.reload=true
+$ {jbossHomeName}/bin/standalone.sh --server-config=standalone-microprofile.xml &
```

Comment:
You can cherrypick the first commit from https://github.com/wildfly/quickstart/pull/423 and use 

```
:standalone-server-type: microprofile
...
include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
```

Reply:
Update: no need to cherry-pick, already merged.

Reply:
@pferraro can you please go for @rhusar suggestion, aligning with all other MP QS after updates?

Reply:
Done.